cpp/python/CMakeLists.txt

```diff
@@ -68,13 +135,23 @@ endif()
 
 include(GNUInstallDirs)
 add_library(ucxx::python ALIAS ucxx_python)
-install(DIRECTORY ${UCXX_SOURCE_DIR}/python/include/ucxx
+install(DIRECTORY ${ucxx_dir}/python/include/ucxx
                   DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-                  COMPONENT python
-                  EXCLUDE_FROM_ALL
+)
+install(TARGETS ucxx_python DESTINATION ${CMAKE_INSTALL_LIBDIR} EXPORT ucxx-python-exports)
+
+rapids_export(
+  INSTALL ucxx-python
+  EXPORT_SET ucxx-python-exports
+  GLOBAL_TARGETS python
+  NAMESPACE ucxx::
 )
```

Comment:
```suggestion
```

Reply:
This seems to be duplicated below.

Reply:
It's not a duplicate. This one is `INSTALL`, while the one below is `BUILD`.

Reply:
Oh right, I missed that. Why does `INSTALL` come first?

Reply:
It doesn't matter which comes first. The order is arbitrary. I guess the reason is that `INSTALL` came first in `cpp/CMakeLists.txt`.

Reply:
Huh, I didn't know order was arbitrary, thanks for the info!